tools/osbuild-manifest

```diff
@@ -81,7 +111,11 @@ def main():
     pipeline = load_manifest(args)
 
     if args.cmd == "info":
-        print_pipeline(pipeline, level=0)
+        if args.json:
+            js = pipeline2json(pipeline)
+            sys.stdout.write(json.dumps(js))
```

Comment:
Doesn't really matter, but I prefer  `json.dump(js, sys.stdout)` for these things.